### PR TITLE
usable with ssh keys now

### PR DESCRIPTION
**Pull request by Lyr3x**

### CODE REVIEW

**Comment by Nesurion** [comment]
This is deprecated right? See #95 ?

**Comment by Lyr3x** [comment]
Yes

Nesurion <notifications@github.com> schrieb am Fr., 17. Juli 2015 00:05:

File: config/ssh.json.example [hunk 1]
```diff
@@ -1,5 +0,0 @@
-{
-  "host" : "0.0.0.0",
-  "user" : "username",
-  "password" : "password"
```
Comment (Nesurion) [inline config/ssh.json.example]: remove the password part

Comment (Nesurion) [inline config/ssh.json.example]: and add the command thing:
"command" : ["echo hi"]

**Pull request closed without merging**